src/js/components/Graphs/NodeInfo/nodesInfo.jsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import renderjson from 'renderjson';
+import { NodeIndoContainer } from './nodeInfo.style';
+
+renderjson.set_icons('+', '-');
+renderjson.set_show_to_level(1);
+
+const NodeInfo = ({ nodeJson }) => {
+  useEffect(() => {
+    const nodeInfoContainer = document.getElementById('nodeInfo');
```

Comment:
`nodeInfoContainer` is the same element you want to render in this component? Thus I think you should use a ref here